patches/minecraft/net/minecraft/nbt/ListTag.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/nbt/ListTag.java
++++ b/net/minecraft/nbt/ListTag.java
+@@ -224,7 +_,7 @@
+    public long[] m_177991_(int p_177992_) {
+       if (p_177992_ >= 0 && p_177992_ < this.f_128716_.size()) {
+          Tag tag = this.f_128716_.get(p_177992_);
+-         if (tag.m_7060_() == 11) {
++         if (tag.m_7060_() == 12) {
```

Comment:
Please add a marker comment, prefixed with `FORGE:`, as to what this change does. Additionally, this should probably use the ID constant `Tag.TAG_LONG_ARRAY` to remove the seemingly-magic constant.
```suggestion
+         if (tag.m_7060_() == Tag.TAG_LONG_ARRAY) { // FORGE: Fix MC-260378 by using correct constant for long array tag
```

Reply:
One benefit to including the number 12 is it will be easier to recognize once this issue is fixed by Mojang, preventing forge from accidently patching 12 to Tag.TAG_LONG_ARRAY in the future. Perhaps its worth using one and mentioning the other in the comment

Reply:
Yeah I was not sure whether I should use `Tag.TAG_LONG_ARRAY` in the first place. To add it in the comment is probably the best way.

Reply:
If 12 is used, the comment should be placed on the line before it as its own line. This would make the patch easier to spot as when it is fixed by Mojang, only the comment will remain in the patch hunk as an addition. Either way, it will probably be caught if they ever get around to fixing it, or it might get missed but is harmless.

Reply:
Ok, moved the comment up by one line.